dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/handler/DssRestHandler.java

```diff
@@ -70,12 +70,26 @@ private static DssRestRequest dssRestRequest(HttpRequest request, String content
         final DssRestContentType contentType = DssRestContentType.fromText(request.headers().get("content-Type"));
         final String uri = request.uri();
 
-        return DssRestRequest
+        String boundary = null;
+        if (contentType == DssRestContentType.MULTIPART_FORM_DATA) {
+          boundary = content.substring(0, content.indexOf("\r\n"));
+        }
+
+        return boundary == null ? DssRestRequest
+                .builder()
+                .methodType(methodType)
+                .contentType(contentType)
+                .uri(uri)
+                .content(content)
+                .build()
+                :
+                DssRestRequest
                 .builder()
                 .methodType(methodType)
                 .contentType(contentType)
                 .uri(uri)
                 .content(content)
+                .boundary(boundary)
                 .build();
```

Comment:
```suggestion
        return DssRestRequest
                .builder()
                .methodType(methodType)
                .contentType(contentType)
                .uri(uri)
                .content(content)
                .boundary(boundary)
                .build();
```
boundary 값이 Null일 때를 boundary가 null인 DssRestRequest객체가 만들어지며,
Null이 아닌 값을 가질 경우 boundary에 값이 들어가는 객체가 만들어져서 코드가 중복이 되네요!
위에 방식으로 작성하는 것이 좋아보입니다.

Reply:
저도 null 인 경우를 구분해서 작성할 필요는 없어보이네요.

Reply:
동의합니다! null일 때 boundary에 다른 특별한 값이 들어가지 않는 한 구분할 필요는 없을거 같아요.